frontstage/views/surveys/add_survey.py

```diff
@@ -16,23 +16,41 @@
 
 @surveys_bp.route("/add-survey", methods=["GET", "POST"])
 @jwt_authorization(request)
-def add_survey(_):
+def add_survey(session):
     form = EnrolmentCodeForm(request.form)
 
+    expires_at = session.get_formatted_expires_in()
+
     if request.method == "POST" and form.validate():
         enrolment_code = request.form.get("enrolment_code").lower()
         logger.info("Enrolment code submitted when attempting to add survey", enrolment_code=enrolment_code)
         # Validate the enrolment code
         try:
             iac = iac_controller.get_iac_from_enrolment(enrolment_code)
             if iac is None:
-                logger.info("Enrolment code not found when attempting to add survey", enrolment_code=enrolment_code)
+                logger.info(
+                    "Enrolment code not found when attempting to add survey",
+                    enrolment_code=enrolment_code,
+                    expires_at=expires_at,
```

Comment:
Not sure how useful adding the expires_at in the logs would be, by virtue of the jwt_authorization decorator it will always be valid

Reply:
This has been removed. Not sure why I put these in!